### ISSUE TITLE
EasyCrypt mode is broken

### ISSUE THREAD

**Issue by strub** [ProofGeneral/PG#167]
Hi. Starting with commit 34d3aaa272e4bc38fbcb59b35f5cb8984a223a31, EasyCrypt mode stucks after having processed the first command. The error messages in the *Messages* buffer are:

```
error in process filter: condition-case: Symbol’s function definition is void: nil
error in process filter: Symbol’s function definition is void: nil
```

I don't know how to get Emacs to print the backtrace. Any help is welcome :)

**Comment by Matafou** [comment]
This seems to be my fault. I will fix it quickly. Sorry for that.

P

Le lun. 6 mars 2017 à 09:23, Pierre-Yves Strub <notifications@github.com> a
écrit :

**Comment by Matafou** [comment]
Should be fixed by 62ec846f. Please reopen if not.
